polaris-react/src/components/TopBar/components/UserMenu/UserMenu.scss

```diff
@@ -4,7 +4,26 @@
   max-width: 160px;
   margin-right: var(--p-space-2);
 
+  #{$se23} & {
+    margin-right: 0;
+    padding: 0 var(--p-space-2) 0 calc(var(--p-space-5) / 2);
```

Comment:
just hardcoding 10px with a disable comment here is fine instead of doing the `calc`